odoo/cli/tsconfig.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+import glob
+import itertools
+import os
+import sys
+import re
+import json
+
+from . import Command
+from odoo.modules.module import MANIFEST_NAMES
+
+
+class TSConfig(Command):
+    """Generates tsconfig files for javascript code"""
+
+    def __init__(self):
+        self.command_name = "tsconfig"
+
+    def get_module_list(self, path):
+        mods = itertools.chain.from_iterable(
+            glob.glob(os.path.join(path, '*/%s' % mname))
+            for mname in MANIFEST_NAMES
+        )
+        return [mod.split(os.path.sep)[-2] for mod in mods]
+
+    def clean_path(self, path):
+        regex = r"//+"
```

Comment:
```suggestion
        regex = r"/{2,}"
```
seems clearer at first glance, though I'll let you be the judge.